docs/HardwareAsAService.md

```diff
@@ -16,8 +16,9 @@ Here you can find a non-exhaustive list of companies that provide such a service
 - [RaspiBlitz](https://shop.fulmo.org/raspiblitz/)
 - [Umbrel](https://getumbrel.com/)
 
-:: Warning
-BTCPay Server cannot answer questions relating to integrations made by third-party entities. Please report bugs and issues directly to the entity that provided you with BTCPay Server inside their product.
-::
+:::danger
```

Comment:
Please do not use `dangerous` unless something is really dangerous. In this case a blue mark is absolutely fine. If we keep using dangerous often, people may ignore it.

Reply:
Good point.

By blue mark, you mean :::tip ?

Reply:
Personally, I don't think it's a warning either. It's simply a prominent indicator that officially something isn't supported.